src/evo/assetlocktx.cpp

```diff
@@ -113,13 +114,18 @@ bool CAssetUnlockPayload::VerifySig(const llmq::CQuorumManager& qman, const uint
     // and at the quorumHash must be active in either the current or previous quorum cycle
     // and the sig must validate against that specific quorumHash.
 
+
     Consensus::LLMQType llmqType = Params().GetConsensus().llmqTypePlatform;
 
-    // We check at most 2 quorums
-    const auto quorums = qman.ScanQuorums(llmqType, pindexTip, 2);
+    const auto& llmq_params_opt = Params().GetLLMQ(llmqType);
+    assert(llmq_params_opt.has_value());
+
+    // after deployment WITHDRAWALS activated we check not to quorum, but all active quorums + 1 the latest inactive
+    const int quorums_to_scan = DeploymentActiveAt(*pindexTip, Params().GetConsensus(), Consensus::DEPLOYMENT_WITHDRAWALS) ? (llmq_params_opt->signingActiveQuorumCount + 1) : 2;
```

Comment:
could use `keepOldConnections` here
```suggestion
    const int quorums_to_scan = DeploymentActiveAt(*pindexTip, Params().GetConsensus(), Consensus::DEPLOYMENT_WITHDRAWALS) ? llmq_params_opt->keepOldConnections : 2;
```

Reply:
> could use keepOldConnections here

technically, yes

But logically I can't explain myself how  `signingActiveQuorumCount` and `keepOldConnections` are related.

Reply:
```
    // Used for intra-quorum communication. This is the number of quorums for which we should keep old connections.
    // For non-rotated quorums it should be at least one more than the active quorums set.
    // For rotated quorums it should be equal to 2 x active quorums set.
    int keepOldConnections;
```
https://github.com/dashpay/dash/blob/develop/src/llmq/params.h#L102-L105

i.e. active set + 1 quorum that became inactive recently, exactly what you need here

Reply:
I'd still says, that in some particular cases it is matched with "+1" indeed, but I can't justify use "keep old connections" as amount of quorums which makes signature valid; also "2x active quorums" for rotating is not relevant for this situation.